scripts-components/toggler/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, EventListeners, DelegateEvent } from '@mangoweb/scripts-base'
+
+// data-target
+// data-class-name
+// data-toggle-others
+// data-prevent-default
+
+export default class Toggler extends Component<void> {
```

Comment:
To `<void>` není třeba. Ideálně bych ale přijímal to `.toggler` jako prop, stejně jako `.is-active`.